### PR TITLE
feat(an): Bootstrap application for accident notification

### PR DESCRIPTION
**Pull request by Thorrik58**
# Bootstrap Accident Notification application

https://app.clickup.com/t/m7arff

## What

Bootstrap the start of the accident notification application.

## Why

Needed for accident notification application.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/accident-notification/src/lib/AccidentNotificationTemplate.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  translationNamespaces: [
+    ApplicationConfigurations.AccidentNotification.translation,
+  ],
+  dataSchema,
+  stateMachineConfig: {
+    initial: AccidentNotificationStates.draft,
+    states: {
+      [AccidentNotificationStates.draft]: {
+        meta: {
+          name: 'hello',
```
Comment (baering) [inline libs/application/templates/accident-notification/src/lib/AccidentNotificationTemplate.ts:47]: 🤓

**Review by baering — COMMENTED** [review]
You'll want to run ` yarn nx format:write` on your branch and commit the changes.

Will revisit when there is a green build.

File: libs/application/templates/accident-notification/src/lib/AccidentNotificationTemplate.ts (hunk 1 above)
Comment (Thorrik58) [reply libs/application/templates/accident-notification/src/lib/AccidentNotificationTemplate.ts:47]: Oops thanks

**Review by pshomov — APPROVED** [review]
LGTM

**Review by jeremybarbet — APPROVED** [review]
LGTM

**Pull request merged**